tox.ini

```diff
@@ -1,6 +1,10 @@
+[tox]
+envlist = lint, test, srclint
+minversion = 3.25.0
+skipsdist = True
+
 [testenv]
-deps =
-        -r requirements.txt
+basepython = python3.6
```

Comment:
I tried it but I saw that the upgrade is non-trivial as it also implies upgrading the Dockerfiles:
currently they are based on `centos:7` but that image is deprecated in favor of `quay.io/centos/centos:stream[8,9]`.
This needs a thorough testing but I agreed with @nb-ohad to have a first iteration for consumer/provider addon ASAP so IMHO this can be done on a 2nd iteration.